src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlRetry/SqlRetryServiceOptions.cs

```diff
@@ -24,7 +24,7 @@ public class SqlRetryServiceOptions
         public const string SqlServer = "SqlServer";
 
         /// <summary>Maximum number of (re)try attempts on a retriable error.</summary>
-        public int MaxRetries { get; set; } = 5;
+        public int MaxRetries { get; set; } = 100;
```

Comment:
Isn't 100 to high?
If an operation is failing more than 10 times in a row, I believe there is something more critical happening. 
My only concern is that we can hide a more serious and complex problem setting max retries to 100.

Reply:
These are truly retriable exceptions. Consider backend process. It should never fail because of retriable exception. Hence high number. This is different from retrying blindly.